Shared/Supporting Files/Extensions/View+Sheet.swift

```diff
@@ -176,28 +179,33 @@ private extension SheetModifier {
     func makeContentWithSheetWrapper(_ content: Content) -> some View {
         ZStack {
             content
-                .popover(
-                    isPresented: Binding(
-                        get: { isPresented && !isSheetLayout },
-                        set: { isPresented = $0 }
-                    )
-                ) {
+                .task(id: isPresented) {
+                    if isPresented {
+                        // Sleep to prevent appearing when other content is disappearing.
+                        try? await Task.sleep(nanoseconds: 1000)
```

Comment:
Generally, putting fixes with time will has other side effects and may manifest differently on different devices and situations. I'm ok with the change if you and @yo1995 are ok with this and have done enough testing.

Reply:
I agree. Using sleep is generally speaking an anti-pattern. Ideally this extension can be refactored but it would require effort that doesn't have much value.

I'm OK with using delay to mitigate the problem temporarily.

Reply:
From my testing, it seems to work as expected. I will leave it in for now as a temporary solution.